frontend/src/components/Psychologist/Reimbursement/Billing.jsx

```diff
@@ -1,8 +1,50 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { HashLink } from 'react-router-hash-link';
 
-const Billing = ({ total }) => {
-  useEffect(() => {}, []);
+import MonthPicker from 'components/Date/MonthPicker';
+
+import agent from 'services/agent';
+import { formatMonth } from 'services/date';
+
+import BillingTable from './BillingTable';
+
+import styles from './billing.cssmodule.scss';
+
+const Billing = () => {
+  const [month, setMonth] = useState({
+    year: new Date().getFullYear(),
+    month: new Date().getMonth() + 1,
+  });
+
+  const [valuesByDate, setValuesByDate] = useState({ appointments: {}, patients: {} });
+
+  useEffect(() => {
+    agent.Appointment.get().then(response => {
+      const appointmentsByDate = {};
+      const patientsByDate = {};
+      response.appointments.forEach(appointment => {
+        const existingAppointments = appointmentsByDate[appointment.appointmentDate];
+        appointmentsByDate[appointment.appointmentDate] = existingAppointments ? existingAppointments + 1 : 1;
```

Comment:
I don't understand these lines, why increment the number of appointments per date?

Reply:
because that's what we want to count and display in the bill table

Reply:
yeah my bad I didn't understand the code earlier but now it's fine :)